libkbfs/block_retrieval_queue.go

```diff
@@ -179,8 +179,8 @@ func (brq *blockRetrievalQueue) Request(ctx context.Context, priority int, kmd K
 				brq.config.BlockCache().GetWithPrefetch(ptr)
 			if err == nil && cachedBlock != nil {
 				block.Set(cachedBlock, brq.config.codec())
-				brq.triggerPrefetchAfterBlockRetrieved(
-					cachedBlock, kmd, priority, hasPrefetched)
+				brq.Prefetcher().PrefetchAfterBlockRetrieved(
```

Comment:
I see that CI found a couple test hangs, maybe your `TogglePrefetcher` change below wasn't enough and this still needs to be a `go` invocation for some reason?